workflows/nanopore.nf

```diff
@@ -58,11 +59,12 @@ include { PLOT_MOSDEPTH_REGIONS as PLOT_MOSDEPTH_REGIONS_AMPLICON } from '../mod
 //
 // SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
 //
-include { INPUT_CHECK         } from '../subworkflows/local/input_check'
-include { PREPARE_GENOME      } from '../subworkflows/local/prepare_genome_nanopore'
-include { SNPEFF_SNPSIFT      } from '../subworkflows/local/snpeff_snpsift'
-include { VARIANTS_LONG_TABLE } from '../subworkflows/local/variants_long_table'
-include { FILTER_BAM_SAMTOOLS } from '../subworkflows/local/filter_bam_samtools'
+include { INPUT_CHECK                   } from '../subworkflows/local/input_check'
+include { PREPARE_GENOME                } from '../subworkflows/local/prepare_genome_nanopore'
+include { SNPEFF_SNPSIFT                } from '../subworkflows/local/snpeff_snpsift'
+include { VARIANTS_LONG_TABLE           } from '../subworkflows/local/variants_long_table'
+include { FILTER_BAM_SAMTOOLS           } from '../subworkflows/local/filter_bam_samtools'
+include { BAM_VARIANT_DEMIX_BOOT_FREYJA } from '../subworkflows/nf-core/bam_variant_demix_boot_freyja/main'
```

Comment:
@Joon-Klaps we don't seem to be actually running `BAM_VARIANT_DEMIX_BOOT_FREYJA` in this workflow? Is there a reason for that?